tests/history_test.go

```diff
@@ -779,6 +788,8 @@ type ActionsRemoveCommentRequest struct {
 
 type ActionsProcessCheckResultRequest struct {
 	Type         string `json:"type"`
+	Host         string `json:"host,omitempty"`    // Set either Host or Service, not both.
+	Service      string `json:"service,omitempty"` // Set either Host or Service, not both.
 	Filter       string `json:"filter"`
```

Comment:
That comment pretty much applies to `Filter` as well, you want to set exactly one of the three. So I'd visually separate them from the others by an empty line and then add a single comment above that block.